dev_tools/clang_format.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2023 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script formats all C/C++ files in the magma repo using clang-format-11.
+# The script can be executed on the dev VM or on the
+# host machine if clang-format-11 is installed.
+
+set -eou pipefail
+
+CLANG_FILE=/usr/bin/clang-format-11
+if test ! -f "$CLANG_FILE"; then
+  echo "clang-format-11 not found. You should execute the script on the dev VM
+    or install clang-format-11 on your host machine."
+  exit 1
+fi
+
+for dir in /orc8r/gateway/c/ /lte/gateway/c/ /lte/gateway/python/;
+do
+  FILES=$(find "${MAGMA_ROOT}/${dir}" \( -iname "*.c" -o -iname "*.cpp" -o -iname "*.h" -o -iname "*.hpp" \) -print)
```

Comment:
The slash between `${MAGMA_ROOT}/${dir}` is too much, the `dir` already start with a slash`.

Reply:
Imo it makes it easier to parse as a path when reading the code though.

Reply:
There are two options: either remove the slash here or in the list in line 26 at the beginning of each path. Double paths work on Linux usually, but I would not want to keep it this way.

Reply:
Yes, I would favour the latter option.

Reply:
Done, removed the slash in L26